USAGE.md

```diff
@@ -5,14 +5,21 @@
 	3. gradle shadowJar
 	4. java -jar ./build/libs/mike-all.jar (for GUI) or java -jar ./build/libs/mike-all.jar (for CLI)
```

Comment:
I think this is supposed to be:

**java -jar ./build/libs/mike-all.jar** (for GUI) or **java -jar ./build/libs/mike-all.jar cli** (for CLI)

It might also be nice to include the bold typeface to make the commands stand out more? Personal opinion though.

Reply:
Ah yea oops, really good catch